node/mock-toolrunner.ts

```diff
@@ -210,7 +216,17 @@ export class ToolRunner extends events.EventEmitter {
             this.emit('stdout', res.stdout);
 
             if (!ops.silent) {
-                ops.outStream.write(res.stdout + os.EOL);
+                if(this.pipeOutputToTool) {
+                    var pipeToolArgString = this.pipeOutputToTool.args.join(' ') || '';
+                    var pipeToolCmdString = this.ignoreTempPath(this.pipeOutputToTool.toolPath);
+                    if(pipeToolArgString) {
+                        pipeToolCmdString += (' ' + pipeToolArgString);
+                    }
+
+                    cmdString += ' | ' + pipeToolCmdString;
+                }
+
+                ops.outStream.write('[command]' + cmdString + os.EOL);
```

Comment:
This should be ops.outStream.write(res.stdout + os.EOL)?
That's the redline to the left.